src/docker-osx-dev

```diff
@@ -353,22 +353,27 @@ function check_for_shared_folders {
 
   if [[ ! -z "$vbox_shared_folders" ]]; then
     log_error "Found VirtualBox shared folders on your Boot2Docker VM. These may void any performance benefits from using docker-osx-dev:\n$vbox_shared_folders"
-    log_instructions "Would you like this script to remove them?"
-
-    local choice=""
-    select choice in "yes" "no"; do
-      case $REPLY in
-        y|Y|yes|Yes )
-          umount_vboxsf_mounted_folder "$vbox_shared_folders"
-          break
-          ;;
-        n|N|no|No )
-          log_instructions "Please remove the VirtualBox shares yourself and re-run this script. Exiting."
-          exit 1
-          ;;
-      esac
-    done
-  fi
+
+    if [[ "$remove_shared_folders" = true ]]; then
```

Comment:
Hmm, that seems to largely defeat the point of `local` (what happened to "principle of least surprise?"), so I would prefer to pass the variable around explicitly to make the code easier to follow.

I'm not sure refactoring just to avoid passing a variable around is worth it.